src/utils/index.ts

```diff
@@ -121,8 +122,8 @@ export const decodeFocuses = (code: string): Focuses => decode<Focuses>(code, ST
 
 export const encodeFocuses = (focuses: Focuses): string => encode<Focuses>(focuses, STATE_IDS, FOCUSES, Focus.No);
 
-export const alternatingCaseToGraphicProps = (alternatingCase: string) => {
-  const graphicProps = acto(alternatingCase);
+export const alternatingCaseToGraphicProps = (alternatingCase: string): GraphicProps => {
```

Comment:
@drzax I'm curious what you'd do here. There's about 10 different props coming in. Is this alright?